lab/skel_graph/template.cpp

```diff
@@ -15,50 +15,50 @@ using namespace std;
 
```

Comment:
I think it may be more appropiate to use std::vector instead of vector. Also, for cin and cout.

Reply:
Yes, sure. Should we keep `using namespace std;` or should we remove it? I propose to remove it everywhere.